### PR TITLE
Orion-15.17.0-SUP-20420

### PR DESCRIPTION
**Pull request by Adirv**

### CODE REVIEW

File: alpha/apps/kaltura/lib/kObjectCreatedHandler.php [hunk 1]
```diff
@@ -21,6 +28,17 @@ public function shouldConsumeCreatedEvent(BaseObject $object)
 	 */
 	public function objectCreated(BaseObject $object)
 	{
+
+        if($object instanceof KuserKgroup)
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/kObjectCreatedHandler.php]: mind indetation

File: alpha/apps/kaltura/lib/batch2/kFlowHelper.php [hunk 2]
```diff
@@ -3231,4 +3231,14 @@ public static function handleLiveToVodFinished(BatchJob $dbBatchJob, kLiveToVodJ
         return $dbBatchJob;
 
     }
+
+    public static function handleKuserKgroupStatusUpdate($kuserkgroup)
+    {
+        if ($kuserkgroup->getStatus() == KuserKgroupStatus::DELETED) {
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/batch2/kFlowHelper.php]: { in new row

File: alpha/apps/kaltura/lib/kObjectCreatedHandler.php (hunk 1 above)
Comment (Adirv) [reply alpha/apps/kaltura/lib/kObjectCreatedHandler.php]: Fixed

File: alpha/apps/kaltura/lib/batch2/kFlowHelper.php (hunk 2 above)
Comment (Adirv) [reply alpha/apps/kaltura/lib/batch2/kFlowHelper.php]: Fixed

**Pull request merged**